kubectl-pg/cmd/addUser.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"log"
+	"strings"
+)
+
+// addUserCmd represents the addUser command
+var addUserCmd = &cobra.Command{
+	Use:   "add-user",
+	Short: "Adds a user to the postgres cluster with given privileges",
+	Long:  `Adds a user to the postgres cluster you can add privileges as well with -p flag.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		clusterName, _ := cmd.Flags().GetString("clusterName")
+		privileges, _ := cmd.Flags().GetString("privileges")
+		if len(args) > 0 {
+			user := args[0]
+			permissions := []string{}
+			perms := []string{}
+			Allowed_Privileges := [...]string{"SUPERUSER", "REPLICATION", "INHERIT", "LOGIN", "NOLOGIN", "CREATEROLE", "CREATEDB", "BYPASSURL"}
```

Comment:
things like that should be [constants](https://tour.golang.org/basics/15) defined at the top of the file

Reply:
Declared at the top as mentioned.